src/main/java/org/folio/utils/LoggingHelper.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.rest.core.models.RequestEntry;
+
+import io.vertx.core.json.JsonObject;
+
+public final class LoggingHelper {
+
+  private static final Logger logger = LogManager.getLogger();
```

Comment:
What class will be written in logs, if it will be LoggingHelper - it will be not very useful for us, we need exact class name

Reply:
The place in the code where actual logging is done will be clear by mesage itself. For example I added to log queries in before RestClient requests and everywhere I used to add method name in the lg message. I'll add method name in other places as well.

Reply:
for example I see similar log, and can quickly understand the class name
![image](https://github.com/folio-org/mod-invoice/assets/25097693/e46d2350-0e46-45ea-b8fe-29cefb5ce686)
For me better to delete this helper, also we will never disable info level, so if statement logger.isInfoEnabled() is not needed

Reply:
done